### PR TITLE
LPD-38643 [Test Fix] ObjectFieldLocalServiceTest

### PR DESCRIPTION
**Pull request by icgsbr**
# Jira Link
- https://liferay.atlassian.net/browse/LPD-38643
# Additional Information
- I'm unable to remove the `Test` prefix, otherwise it will throw `com.liferay.object.exception.ObjectDefinitionNameException$ForbiddenModifiableSystemObjectDefinitionName: Forbidden modifiable system object definition name`
- The validation called in `ObjectDefinitionLocalServiceImpl` on the [_validateName](https://github.com/liferay-bpm/liferay-portal/blob/24ae5874756ae6e1436657aee45bb41fd0581b3c/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java#L2508C3-L2514C4) method comes from the method [isAllowedModifiableSystemObjectDefinitionName](https://github.com/liferay-bpm/liferay-portal/blob/24ae5874756ae6e1436657aee45bb41fd0581b3c/modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java#L33C2-L41C3) in `ObjectDefinitionUtil`

### CODE REVIEW

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
Hey @carolmariaabb 
Could you review this PR, please? :smile:

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:1]: @icgsbr could you please simplify to `"A" + RandomTestUtil.randomString()`? Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java
Comment (icgsbr) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:1]: Hi @carolmariaabb !

According to the validation [isAllowedModifiableSystemObjectDefinitionName](https://github.com/liferay-bpm/liferay-portal/blob/24ae5874756ae6e1436657aee45bb41fd0581b3c/modules/apps/object/object-api/src/main/java/com/liferay/object/definition/util/ObjectDefinitionUtil.java#L33C2-L41C3) it must start with the string "Test". Should I change the validation?

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**